docs/pages/app-signing/apple-team-permissions.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Apple Developer Program roles and permissions for EAS Build
+sidebar_title: Apple Developer Program roles and permissions
+description: Learn about the Apple Developer account membership requirements for creating an EAS Build.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+When creating a build for and iOS device on EAS, an Apple Developer account is required. This account has necessary permissions to submit the app for review, manage app's distribution, and generate required [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles.
+
+If the Apple Developer account is an individual account, only the account owner can generate the app signing credentials. This guide provides steps that the account owner has to follow to ensure app signing credentials are generated. It also provides steps for the team developer to create an EAS Build.
+
+> See [Apple's documentation on Program Roles](https://developer.apple.com/support/roles/) for details on the different roles and their permissions based on the type of Developer account.
+
+## Steps for Apple Developer account owner
+
+The owner of the individual Apple Developer account needs to generate a distribution certificate (if submitting an app to the App Store), provisioning profile, or a push key (if submitting an app to use a push notification service).
+
+Once the owner login to their [Expo account](/accounts/account-types/), they can create or update the provisioning profile by running `eas credentials` using the EAS CLI.
+
+<Terminal cmd={['$ eas login', '', '$ eas credentials']} />
+
+The CLI will prompt for selecting a [build profile](/build/eas-json/#build-profiles) to use for the EAS Build. If the owner is creating a production build, follow the the steps to [create a distribution provisioning profile](/tutorial/eas/ios-production-build/#create-a-distribution-provisioning-profile). To create a developer build, follow the steps to [create a provisioning profile](/tutorial/eas/ios-development-build-for-devices/#provisioning-profile).
+
+This ensures that the provisioning profile associated with the Expo account has necessary permissions.
+
```

Comment:
Worth adding a side not here if you can't use `eas credentials`, something like:

> For projects with existing credentials, see [using existing credentials](https://docs.expo.dev/app-signing/existing-credentials/) on how to sync these to EAS or manage them manually.